src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.split(';').filter((line) => line.trim() !== '');
+
+  lines.forEach((line) => {
+    const parts = line.split(':');
+
+    if (parts.length >= 2) {
+      const property = parts[0].trim();
+      const value = parts.slice(1).join(':').trim();
```

Comment:
I think you don't need slice in this case as you can just get the second element, and why do you need to join ":"?